app/jobs/hook_job.rb

```diff
@@ -24,6 +24,10 @@ def process_slack_integration(hook, event_name, event_data)
   def process_dialogflow_integration(hook, event_name, event_data)
     return unless ['message.created', 'message.updated'].include?(event_name)
 
+    message = event_data[:message]
+
+    return unless message.webhook_sendable?
+
```

Comment:
Let's have this check inside the processor service so that all logic related to Dialogflow will remain encapsulated in a single place.

Reply:
We do have this check here https://github.com/chatwoot/chatwoot/blob/develop/lib/integrations/bot_processor_service.rb#L46